src/app/search/search-results-total-count/search-results-total-count.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-search-results-total-count',
+  templateUrl: './search-results-total-count.component.html',
+  styleUrls: ['./search-results-total-count.component.css']
+})
```

Comment:
change detection strategy could be changed to `OnPush`

https://blog.thoughtram.io/angular/2016/02/22/angular-2-change-detection-explained.html